test/src/models/form_group_test.dart

```diff
@@ -66,7 +66,7 @@ void main() {
     test('FormGroup is invalid if all FormControl are invalid', () {
       final form = FormGroup({
         'name': FormControl(validators: [Validators.required]),
-        'email': FormControl(validators: [Validators.email]),
+        'email': FormControl<String>(validators: [Validators.email]),
```

Comment:
This will not allow creating a control with dynamic type as:
```dart
final control = FormControl(
   value: "I'm a String control",
   validators: [Validators.email], // exception is thrown
);
```
So I suggest leaving the Email validator as **dynamic**